website/docs/d/policy_security_policy_rule.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+subcategory: "Firewall"
```

Comment:
this document is not relevant anymore, since data source is removed?

Reply:
Done.